pages/events/[id].tsx

```diff
@@ -0,0 +1,232 @@
+import { useRouter } from 'next/router';
+import Image from 'next/image';
+import { useGetEventByIdQuery } from 'generated/graphql';
+import { Typography } from 'components/atoms/Typography/Typography';
+import { useState } from 'react';
+import Camera from 'components/icons/Camera.svg';
+import Star from 'components/icons/Star.svg';
+import UserIcon from 'components/icons/UserIcon.svg';
+import Ticket from 'components/icons/Ticket.svg';
+
+const EventDetailsPage = () => {
+	const router = useRouter();
+	const id = router.query.id as string;
+	const { data, loading } = useGetEventByIdQuery({ variables: { id } });
+	const [isEvent, setIsEvent] = useState(true);
+	if (loading) {
+		return (
+			<div>
+				<Typography component="h1">Szczegóły wydarzenia</Typography>
+				<div className="flex justify-center">
+					<Image
+						src={'/RingSpinner.svg'}
+						width={80}
+						height={80}
+						alt="Spinner"
+					/>
+				</div>
+			</div>
+		);
+	}
+	if (!data) {
+		return (
+			<div>
+				<Typography component="h1">Data not found</Typography>
+			</div>
+		);
+	}
+	console.log(data, loading);
+	return (
+		<div>
+			<div>
+				<Typography component="h1">Wydarzenie</Typography>
+			</div>
+			<div>
+				<Image
+					className="h-32 w-full rounded-lg object-cover lg:h-48"
+					src={data.event?.image?.url || '/part.png'}
+					height={600}
+					width={600}
+					alt=""
+				/>
+				<div className="mt-6 sm:mt-2 2xl:mt-5">
+					<div className="border-b border-gray-200">
+						<div className="mx-auto max-w-5xl px-4 sm:px-6 lg:px-8">
+							<nav className="-mb-px flex space-x-8">
+								<button
+									onClick={() => setIsEvent(true)}
+									className={
+										isEvent
+											? 'border-b-2 border-blue-500 text-gray-900'
+											: 'whitespace-nowrap border-b-2 border-transparent py-4 px-1 text-sm font-medium text-gray-500 hover:border-gray-300 hover:text-gray-700'
+									}
+								>
+									Wydarzenie
+								</button>
+								<button
+									onClick={() => setIsEvent(false)}
+									className={
+										isEvent
+											? 'whitespace-nowrap border-b-2 border-transparent py-4 px-1 text-sm font-medium text-gray-500 hover:border-gray-300 hover:text-gray-700'
+											: 'border-b-2 border-blue-500 text-gray-900'
+									}
+								>
+									Budzet
+								</button>
+							</nav>
+						</div>
+					</div>
+				</div>
+				{isEvent ? (
```

Comment:
I think it should be whole another component, just for readability

Reply:
It's separated for two different components events/budget details.